modules/sql-engine/src/integrationTest/sql/types/date/test_date.test

```diff
@@ -0,0 +1,38 @@
+# name: test/sql/types/date/test_date.test
+# description: Test basic DATE functionality
+# group: [date]
+
+statement ok
+PRAGMA enable_verification
+
+# create and insert into table
+statement ok
+CREATE TABLE dates(i DATE)
+
+statement ok
+INSERT INTO dates VALUES ('1993-08-14')
+
+# TODO https://issues.apache.org/jira/browse/IGNITE-15123 Multi-tuple insert fails on validation
+statement ok
+INSERT INTO dates VALUES (NULL)
+
+# check that we can select dates
+query T rowsort
+SELECT * FROM dates
```

Comment:
seems we need ordering definition if we wait defined ordered results ?

Reply:
We have ordering on the test framework side here (`rowsort`), is that not enough?